services/intersection-api/api/src/main/java/us/dot/its/jpo/ode/api/accessors/IntersectionCriteria.java

```diff
@@ -0,0 +1,59 @@
+package us.dot.its.jpo.ode.api.accessors;
+
+import org.springframework.data.mongodb.core.query.Criteria;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.time.Instant;
+import java.util.Date;
+
+public class IntersectionCriteria extends Criteria {
+
+    /**
+     * Build a query criteria object based on a time window
+     *
+     * @param fieldName        the db field to apply criteria to
+     * @param startEpochMillis the nullable start time of the window, in
+     *                         milliseconds since epoch
+     * @param endEpochMillis   the nullable end time of the window, in milliseconds
+     *                         since epoch
+     * @return the criteria object to use for querying
+     */
+    public IntersectionCriteria withinTimeWindow(
+            @Nonnull String fieldName,
+            @Nullable Long startEpochMillis,
+            @Nullable Long endEpochMillis,
+            @Nullable boolean formatAsString) {
```

Comment:
chore(non-blocking): primitive types cannot be annotated (they also cannot be `null`). let's remove the annotation
```suggestion
            boolean formatAsString) {
```

Reply:
yeah, I intended to make this nullable... but that's not necessary. Removed annotation